src/types/post.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from './user';
```

Comment:
Please fix this problem
![image](https://user-images.githubusercontent.com/96409563/197381347-61497980-ccc8-4996-927b-a3b147963e41.png)

Reply:
I don`t have this line in my code. I made screen of a test result, but i don`t understand what`s wrong with titles
<img width="804" alt="image" src="https://user-images.githubusercontent.com/106486742/197406259-63b11419-ba33-4837-b18e-e47e52b3a4f1.png">
<img width="685" alt="image" src="https://user-images.githubusercontent.com/106486742/197406296-8f0a8d43-5df9-4435-b634-8a81c0f20ec2.png">